src/test/java/org/opentripplanner/model/TimetableTest.java

```diff
@@ -170,24 +150,25 @@ public void testUpdate() {
     );
     tripUpdate = tripUpdateBuilder.build();
     assertEquals(20 * 60, timetable.getTripTimes(trip_1_1_index).getArrivalTime(2));
-    tripTimesPatch = timetable.createUpdatedTripTimes(tripUpdate, timeZone, serviceDate);
-    updatedTripTimes = tripTimesPatch.getTripTimes();
-    assertNotNull(updatedTripTimes);
+    patch = timetable.createUpdatedTripTimes(tripUpdate, timeZone, serviceDate);
+    var updatedTripTimes = patch.getTripTimes();
+    Assertions.assertNotNull(updatedTripTimes);
     timetable.setTripTimes(trip_1_1_index, updatedTripTimes);
     assertEquals(20 * 60 + 120, timetable.getTripTimes(trip_1_1_index).getArrivalTime(2));
 
     //---
 
-    spt =
+    /*spt =
       AStarBuilder
         .oneToOne()
         .setContext(new RoutingContext(options, graph, stop_a, stop_c))
         .getShortestPathTree();
 
     path = spt.getPath(stop_c);
-    assertNotNull(path);
+    Assertions.assertNotNull(path);
     endTime = startTime + 20 * 60 + 120;
-    assertEquals(endTime, path.getEndTime());
+    Assertions.assertEquals(endTime, path.getEndTime());
+     */
 
     // cancel trip
```

Comment:
Actually I think we could remove this part of the test and remove the whole CANCELED handling from createUpdatedTripTimes as it isn't actually being used. We only handle SCHEDULED updated with the createUpdatedTripTimes method and we could state that its the only use for the method.

Reply:
Tests removed.